crates/bevy_math/Cargo.toml

```diff
@@ -36,7 +36,7 @@ bevy_math = { path = ".", version = "0.15.0-dev", features = ["approx"] }
 glam = { version = "0.28", features = ["approx"] }
 
 [features]
-default = ["rand", "bevy_reflect"]
+default = ["rand", "bevy_reflect", "libm"]
```

Comment:
I think this will be a very controversial change.

Other than that this PR looks good to me now. (But I want someone else to take a thorough look too, because I'm not sure I haven't missed something.)

Reply:
We should measure the performance impact

Reply:
```suggestion
default = ["rand", "bevy_reflect"]
```